[PATCH] TOMOYO: Add missing policy load call.

Commit efe836ab "TOMOYO: Add built-in policy support." introduced
tomoyo_load_builtin_policy() but was by error called from nowhere.
As a result, builtin-policy cannot be loaded. Also, TOMOYO cannot be activated
if CONFIG_SECURITY_TOMOYO_OMIT_USERSPACE_LOADER=y.

Signed-off-by: Tetsuo Handa <penguin-kernel@I-love.SAKURA.ne.jp>
---
 security/tomoyo/securityfs_if.c |    1 +
 1 file changed, 1 insertion(+)

--- linux-3.1.0.orig/security/tomoyo/securityfs_if.c
+++ linux-3.1.0/security/tomoyo/securityfs_if.c
@@ -147,6 +147,7 @@ static int __init tomoyo_initerface_init
 			    TOMOYO_MANAGER);
 	tomoyo_create_entry("version",          0400, tomoyo_dir,
 			    TOMOYO_VERSION);
+	tomoyo_load_builtin_policy();
 	return 0;
 }
 
